Development/nmos-cpp-node/node_implementation.cpp

```diff
@@ -1236,3 +1237,11 @@ nmos::experimental::node_implementation make_node_implementation(nmos::node_mode
         .on_validate_channelmapping_output_map(make_node_implementation_map_validator()) // may be omitted if not required
         .on_channelmapping_activated(make_node_implementation_channelmapping_activation_handler(gate));
 }
+
+// This constructs all the callbacks including OCSP stapling used to integrate the example device-specific underlying implementation
+// into the server instance for the NMOS Node.
+nmos::experimental::node_implementation make_node_implementation(nmos::node_model& model, nmos::experimental::ocsp_state& ocsp_state, slog::base_gate& gate)
+{
+    return make_node_implementation(model, gate)
+        .on_get_ocsp_response(nmos::make_ocsp_response_handler(ocsp_state, gate));
```

Comment:
How about just doing this in _main.cpp_, to really show how separable OCSP is from the rest of the core NMOS bits? Could be a nice model for when we come to add Auth/Cert Prov in the same way?

Especially since we're adding `nmos::ocsp_behaviour_thread` in _main.cpp_...